server/assets-v1/templates/src/pages/client_portal/register.html

```diff
@@ -230,6 +221,14 @@ <h1 class="font-bold text-3xl md:text-4xl text-[#3751FE] mb-11 animate-slideFrom
           return false;
         }
       };
+      const showToastMessage = (message, type) => {
```

Comment:
I notice here that you use the same function `showToastMessage` as you do in the profile.html page. Not for this PR, but I wonder if we should explore putting this message in a simple library of utils. Notice that on line 152, you import such things as `ref` from Vue. I know we're using a CDN, but I wonder could we still have our own libraries?

Reply:
Yes, we can import showToastMessage function from utils folder. We can implement it. Maybe, I will create extra issue for this.